include/rmm/cuda_device.hpp

```diff
@@ -43,6 +43,35 @@ struct cuda_device_id {
   /// @briefreturn{The wrapped integer value}
   [[nodiscard]] constexpr value_type value() const noexcept { return id_; }
 
+  // TODO re-add doxygen comment specifier /** for these hidden friend operators once this Breathe
+  // bug is fixed: https://github.com/breathe-doc/breathe/issues/916
+  //! @conda Doxygen_Suppress
```

Comment:
D'oh. Muscle memory typo.

```suggestion
  //! @cond Doxygen_Suppress
```